examples/Example_OLR_Tracking_model/Example_OLR_Tracking_model.ipynb

```diff
@@ -79,62 +78,63 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_out='../'"
+    "data_out=Path('../')"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "start downloading data\n",
+      "start extracting data\n",
+      "data extracted\n"
+     ]
+    }
+   ],
    "source": [
-    "# # Download the data: This only has to be done once for all tobac examples and can take a while\n",
-    "# file_path='https://zenodo.org/record/3195910/files/climate-processes/tobac_example_data-v1.0.1.zip'\n",
-    "# #file_path='http://zenodo..'\n",
-    "# tempfile='temp.zip'\n",
-    "# print('start downloading data')\n",
-    "# request=urllib.request.urlretrieve(file_path,tempfile)\n",
-    "# print('start extracting data')\n",
-    "# shutil.unpack_archive(tempfile,data_out)\n",
-    "# #zf = zipfile.ZipFile(tempfile)\n",
-    "# #zf.extractall(data_out)\n",
-    "# os.remove(tempfile)\n",
-    "# print('data extracted')\n"
+    "# Download the data: This only has to be done once for all tobac examples and can take a while\n",
+    "data_file = list(data_out.rglob('data/Example_input_OLR_model.nc'))\n",
+    "if len(data_file) == 0:\n",
+    "    file_path='https://zenodo.org/record/3195910/files/climate-processes/tobac_example_data-v1.0.1.zip'\n",
+    "    #file_path='http://zenodo..'\n",
+    "    tempfile=Path('temp.zip')\n",
+    "    print('start downloading data')\n",
+    "    request=urllib.request.urlretrieve(file_path, tempfile)\n",
+    "    print('start extracting data')\n",
+    "    shutil.unpack_archive(tempfile, data_out)\n",
```

Comment:
Nice, in my example I was not aware that `shutil` is now also compatible with `pathlib.Path` objects

Reply:
Yes, we should certainly remove Python 2 support, and I'm also in favor of limiting to >=3.7. In fact the conda-forge recipe is already restricted to >=3.6, so it would be easy to bump that up. We should edit the readme to reflect this

Reply:
Perfect, thanks! I agree with you both on removing Python 2 support